### ISSUE TITLE
backupccl: `SHOW SCHEDULES` on a scheduled created <22.2 errors out

### ISSUE THREAD

**Issue by adityamaru** [cockroachdb/cockroach#87934]
We moved `backup.proto` and consequently `cockroach.ccl.backupccl.ScheduledBackupExecutionArgs` to `cockroach.ccl.backupccl.backuppb.ScheduledBackupExecutionArgs`. `SHOW SCHEDULES` is powered by `crdb_internal.pb_to_json` and so when we run `SHOW SCHEDULES` on a scheduled backup created on a cluster <22.2 we get 
```
root@localhost:26257/defaultdb> show schedules;
ERROR: crdb_internal.pb_to_json(): marshaling cockroach.jobs.jobspb.ExecutionArguments; msg=args:<type_url:"type.googleapis.com/cockroach.ccl.backupccl.ScheduledBackupExecutionArgs" value:"\022-BACKUP INTO 'nodelocal://1/foo' WITH detached" >: unknown message type "cockroach.ccl.backupccl.ScheduledBackupExecutionArgs"
```
because we are looking for the proto in the wrong place.

Jira issue: CRDB-19599

**Comment by adityamaru** [comment]
Remarking as release blocker as discussed offline since this bug not only breaks `SHOW SCHEDULES` but also prevents execution of scheduled jobs on upgrade from pre-22.2 clusters to 22.2. The fix is in the works https://github.com/cockroachdb/cockroach/pull/87991

**Comment by adityamaru** [comment]
Fixed by #87999
